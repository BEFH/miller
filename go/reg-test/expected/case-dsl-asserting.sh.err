
Miller: is_absent type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_absent type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_absent type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_absent type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_absent type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_absent type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_empty type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_empty type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_empty type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_empty type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_empty type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_empty type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_empty type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_empty type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_empty_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_empty_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_empty_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_empty_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_empty_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_empty_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_empty_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_empty_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_non_empty_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_non_empty_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_non_empty_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_non_empty_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_non_empty_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_non_empty_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_non_empty_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_not_empty type-assertion failed at NR=4 FNR=4 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_not_empty type-assertion failed at NR=3 FNR=3 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_not_empty type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_not_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_not_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_not_map type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_not_null type-assertion failed at NR=4 FNR=4 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_not_null type-assertion failed at NR=3 FNR=3 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_not_null type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_not_null type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_not_null type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_null type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_null type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_null type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_null type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_null type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_numeric type-assertion failed at NR=4 FNR=4 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_numeric type-assertion failed at NR=3 FNR=3 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_numeric type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_numeric type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_numeric type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_numeric type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_present type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_present type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_string type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_string type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_string type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_string type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
Miller: is_string type-assertion failed at NR=1 FNR=1 FILENAME=./reg-test/input/nullvals.dkvp
