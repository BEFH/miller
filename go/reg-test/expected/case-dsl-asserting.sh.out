
mlr --opprint put $f=asserting_absent($nosuch) ./reg-test/input/nullvals.dkvp
a x y z
b 1 2 -
b 3 4 -
b 5 - -
b - 6 -
b - - -

mlr --opprint put $f=asserting_absent(@nosuch) ./reg-test/input/nullvals.dkvp
a x y z
b 1 2 -
b 3 4 -
b 5 - -
b - 6 -
b - - -

mlr --opprint put $f=asserting_absent($x) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_absent($y) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_absent($z) ./reg-test/input/nullvals.dkvp

mlr --opprint put @somesuch=1;$f=asserting_absent(@somesuch) ./reg-test/input/nullvals.dkvp

mlr --opprint put foo=asserting_absent($*) ./reg-test/input/nullvals.dkvp

mlr --opprint put foo=asserting_absent({1:2}) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_empty($z) ./reg-test/input/nullvals.dkvp
a x y z f
b 1 2 - -
b 3 4 - -
b 5 - - -
b - 6 - -
b - - - -

mlr --opprint put $f=asserting_empty($x) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_empty($y) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_empty($nosuch) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_empty(@nosuch) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_empty({1:2}) ./reg-test/input/nullvals.dkvp

mlr --opprint put @somesuch=1;$f=asserting_empty(@somesuch) ./reg-test/input/nullvals.dkvp

mlr --opprint put foo=asserting_empty($*) ./reg-test/input/nullvals.dkvp

mlr --opprint put foo=asserting_empty({1:2}) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_empty_map({}) ./reg-test/input/nullvals.dkvp
a x y z
b 1 2 -
b 3 4 -
b 5 - -
b - 6 -
b - - -

mlr --opprint put $f=asserting_empty_map($*) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_empty_map($x) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_empty_map($y) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_empty_map($z) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_empty_map($nosuch) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_empty_map(@nosuch) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_empty_map({1:2}) ./reg-test/input/nullvals.dkvp

mlr --opprint put @somesuch=1;$f=asserting_empty_map(@somesuch) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_map($*) ./reg-test/input/nullvals.dkvp
a x y z f:a f:x f:y f:z
b 1 2 - b   1   2   -
b 3 4 - b   3   4   -
b 5 - - b   5   -   -
b - 6 - b   -   6   -
b - - - b   -   -   -

mlr --opprint put $f=asserting_map({1:2}) ./reg-test/input/nullvals.dkvp
a x y z f:1
b 1 2 - 2
b 3 4 - 2
b 5 - - 2
b - 6 - 2
b - - - 2

mlr --opprint put $f=asserting_map({}) ./reg-test/input/nullvals.dkvp
a x y z
b 1 2 -
b 3 4 -
b 5 - -
b - 6 -
b - - -

mlr --opprint put $f=asserting_map($x) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_map($y) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_map($z) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_map($nosuch) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_map(@nosuch) ./reg-test/input/nullvals.dkvp

mlr --opprint put @somesuch=1;$f=asserting_map(@somesuch) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_nonempty_map($*) ./reg-test/input/nullvals.dkvp
a x y z f:a f:x f:y f:z
b 1 2 - b   1   2   -
b 3 4 - b   3   4   -
b 5 - - b   5   -   -
b - 6 - b   -   6   -
b - - - b   -   -   -

mlr --opprint put $f=asserting_nonempty_map({1:2}) ./reg-test/input/nullvals.dkvp
a x y z f:1
b 1 2 - 2
b 3 4 - 2
b 5 - - 2
b - 6 - 2
b - - - 2

mlr --opprint put $f=asserting_nonempty_map($x) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_nonempty_map($y) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_nonempty_map($z) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_nonempty_map($nosuch) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_nonempty_map(@nosuch) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_nonempty_map({}) ./reg-test/input/nullvals.dkvp

mlr --opprint put @somesuch=1;$f=asserting_nonempty_map(@somesuch) ./reg-test/input/nullvals.dkvp

mlr --opprint put $*=asserting_not_empty($*) ./reg-test/input/nullvals.dkvp
a x y z
b 1 2 -
b 3 4 -
b 5 - -
b - 6 -
b - - -

mlr --opprint put $f=asserting_not_empty($nosuch) ./reg-test/input/nullvals.dkvp
a x y z
b 1 2 -
b 3 4 -
b 5 - -
b - 6 -
b - - -

mlr --opprint put $f=asserting_not_empty(@nosuch) ./reg-test/input/nullvals.dkvp
a x y z
b 1 2 -
b 3 4 -
b 5 - -
b - 6 -
b - - -

mlr --opprint put $f=asserting_not_empty({1:2}) ./reg-test/input/nullvals.dkvp
a x y z f:1
b 1 2 - 2
b 3 4 - 2
b 5 - - 2
b - 6 - 2
b - - - 2

mlr --opprint put $nosuch=asserting_not_empty($nosuch) ./reg-test/input/nullvals.dkvp
a x y z
b 1 2 -
b 3 4 -
b 5 - -
b - 6 -
b - - -

mlr --opprint put @somesuch=1;$f=asserting_not_empty(@somesuch) ./reg-test/input/nullvals.dkvp
a x y z f
b 1 2 - 1
b 3 4 - 1
b 5 - - 1
b - 6 - 1
b - - - 1

mlr --opprint put $*=asserting_not_empty($*) ./reg-test/input/nullvals.dkvp
a x y z
b 1 2 -
b 3 4 -
b 5 - -
b - 6 -
b - - -

mlr --opprint put $*=asserting_not_empty({1:2}) ./reg-test/input/nullvals.dkvp
1
2
2
2
2
2

mlr --opprint put $f=asserting_not_empty($x) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_not_empty($y) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_not_empty($z) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_not_map($x) ./reg-test/input/nullvals.dkvp
a x y z f
b 1 2 - 1
b 3 4 - 3
b 5 - - 5
b - 6 - -
b - - - -

mlr --opprint put $f=asserting_not_map($y) ./reg-test/input/nullvals.dkvp
a x y z f
b 1 2 - 2
b 3 4 - 4
b 5 - - -
b - 6 - 6
b - - - -

mlr --opprint put $f=asserting_not_map($z) ./reg-test/input/nullvals.dkvp
a x y z f
b 1 2 - -
b 3 4 - -
b 5 - - -
b - 6 - -
b - - - -

mlr --opprint put $f=asserting_not_map($nosuch) ./reg-test/input/nullvals.dkvp
a x y z
b 1 2 -
b 3 4 -
b 5 - -
b - 6 -
b - - -

mlr --opprint put $f=asserting_not_map(@nosuch) ./reg-test/input/nullvals.dkvp
a x y z
b 1 2 -
b 3 4 -
b 5 - -
b - 6 -
b - - -

mlr --opprint put @somesuch=1;$f=asserting_not_map(@somesuch) ./reg-test/input/nullvals.dkvp
a x y z f
b 1 2 - 1
b 3 4 - 1
b 5 - - 1
b - 6 - 1
b - - - 1

mlr --opprint put $f=asserting_not_map($*) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_not_map({1:2}) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_not_map({}) ./reg-test/input/nullvals.dkvp

mlr --opprint put @somesuch=1;$f=asserting_not_null(@somesuch) ./reg-test/input/nullvals.dkvp
a x y z f
b 1 2 - 1
b 3 4 - 1
b 5 - - 1
b - 6 - 1
b - - - 1

mlr --opprint put $*=asserting_not_null($*) ./reg-test/input/nullvals.dkvp
a x y z
b 1 2 -
b 3 4 -
b 5 - -
b - 6 -
b - - -

mlr --opprint put $*=asserting_not_null({1:2}) ./reg-test/input/nullvals.dkvp
1
2
2
2
2
2

mlr --opprint put $f=asserting_not_null($x) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_not_null($y) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_not_null($z) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_not_null($nosuch) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_not_null(@nosuch) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_null($z) ./reg-test/input/nullvals.dkvp
a x y z f
b 1 2 - -
b 3 4 - -
b 5 - - -
b - 6 - -
b - - - -

mlr --opprint put $f=asserting_null($nosuch) ./reg-test/input/nullvals.dkvp
a x y z
b 1 2 -
b 3 4 -
b 5 - -
b - 6 -
b - - -

mlr --opprint put $f=asserting_null(@nosuch) ./reg-test/input/nullvals.dkvp
a x y z
b 1 2 -
b 3 4 -
b 5 - -
b - 6 -
b - - -

mlr --opprint put $f=asserting_null($x) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_null($y) ./reg-test/input/nullvals.dkvp

mlr --opprint put @somesuch=1;$f=asserting_null(@somesuch) ./reg-test/input/nullvals.dkvp

mlr --opprint put $*=asserting_null($*) ./reg-test/input/nullvals.dkvp

mlr --opprint put $*=asserting_null({1:2}) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_numeric($x) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_numeric($y) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_numeric($z) ./reg-test/input/nullvals.dkvp

mlr --opprint put $*=asserting_numeric($*) ./reg-test/input/nullvals.dkvp

mlr --opprint put $*=asserting_numeric({1:2}) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_numeric($nosuch) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_present($x) ./reg-test/input/nullvals.dkvp
a x y z f
b 1 2 - 1
b 3 4 - 3
b 5 - - 5
b - 6 - -
b - - - -

mlr --opprint put $f=asserting_present($y) ./reg-test/input/nullvals.dkvp
a x y z f
b 1 2 - 2
b 3 4 - 4
b 5 - - -
b - 6 - 6
b - - - -

mlr --opprint put $f=asserting_present($z) ./reg-test/input/nullvals.dkvp
a x y z f
b 1 2 - -
b 3 4 - -
b 5 - - -
b - 6 - -
b - - - -

mlr --opprint put @somesuch=1;$f=asserting_present(@somesuch) ./reg-test/input/nullvals.dkvp
a x y z f
b 1 2 - 1
b 3 4 - 1
b 5 - - 1
b - 6 - 1
b - - - 1

mlr --opprint put $*=asserting_present($*) ./reg-test/input/nullvals.dkvp
a x y z
b 1 2 -
b 3 4 -
b 5 - -
b - 6 -
b - - -

mlr --opprint put $*=asserting_present({1:2}) ./reg-test/input/nullvals.dkvp
1
2
2
2
2
2

mlr --opprint put $f=asserting_present($nosuch) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_present(@nosuch) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_string($z) ./reg-test/input/nullvals.dkvp
a x y z f
b 1 2 - -
b 3 4 - -
b 5 - - -
b - 6 - -
b - - - -

mlr --opprint put $*=asserting_string($*) ./reg-test/input/nullvals.dkvp

mlr --opprint put $*=asserting_string({1:2}) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_string($x) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_string($y) ./reg-test/input/nullvals.dkvp

mlr --opprint put $f=asserting_string($nosuch) ./reg-test/input/nullvals.dkvp

